tensorflow/lite/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ cmake_dependent_option(TFLITE_ENABLE_NNAPI_VERBOSE_VALIDATION "Enable NNAPI verb
 option(TFLITE_ENABLE_MMAP "Enable MMAP (unsupported on Windows)" ON)
 option(TFLITE_ENABLE_GPU "Enable GPU" OFF)
 option(TFLITE_ENABLE_XNNPACK "Enable XNNPACK backend" ON)
+option(TFLITE_ENABLE_EXTERNAL "Enable External Delegate backend" ON)
```

Comment:
The name doesn't look trivial.
How about TFLITE_ENABLE_EXTERNAL_DELEGATE ?

Reply:
Sure, that works for me.  I was trying to follow the convention for the other similar delegate options.  I agree "EXTERNAL" by itself is vague.  I'll make the change and resubmit.